Sources/SymbolKit/SymbolGraph/Symbol/Symbol.swift

```diff
@@ -402,7 +367,12 @@ extension SymbolGraph.Symbol {
                 // (e.g. "swift.func" instead of just "func"), strip off the language prefix and
                 // try again.
                 let cleanIdentifier = KindIdentifier.cleanIdentifier(identifier)
-                self = Self.lookupIdentifier(identifier: cleanIdentifier) ?? .unknown
+                if let secondParse = Self.lookupIdentifier(identifier: cleanIdentifier) {
```

Comment:
It seems we sometimes call `KindIdentifier.lookupIdentifier` or `KindIdentifier.cleanIdentifier` and sometimes call `Self.lookupIdentifier` or `Self.cleanIdentifier`. I suggest we only use one style here. Personally I prefer the first.

```suggestion
                if let secondParse = KindIdentifier.lookupIdentifier(identifier: cleanIdentifier) {
```